### PR TITLE
WIP add ivy distribution, upgrade to Angular 13

### PR DESCRIPTION
**Pull request by micahstubbs**
## :memo: Description

WIP PR that adds an Angular ivy distribution, as well as upgrading ngx-vis to Angular 13. I followed [this guide](https://update.angular.io/?v=12.0-13.0) to upgrade Angular.

### :dart: Relevant issues
#424 

### :gem: Type of change

- [x] New feature (non-breaking change which adds functionality)

## :vertical_traffic_light: How Has This Been Tested?

`npm run build` succeeds.

`npm test` fails currently. I would appreciate help debugging the test failure. 
## :checkered_flag: Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- N/A  **I have added tests that prove my fix is effective or that my feature works**
- [ ] **New and existing unit tests pass locally with my changes**
- N/A I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: ng-package.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "lib": {
+    "entryFile": "ngx-vis.ts",
+    "flatModuleFile": "ngxvis"
+  },
+  "allowedNonPeerDependencies": ["vis-data", "vis-network", "vis-timeline", "vis-util"]
```
Comment (micahstubbs) [inline ng-package.json:6]: https://github.com/ng-packagr/ng-packagr/blob/master/docs/dependencies.md

`s/whitelistedNonPeerDependencies/allowedNonPeerDependencies`

**Comment by hypery2k** [comment]
thx for your contribution. If you done with your changes feel free to give me a ping. I can publish a snapshot of the NPM package for testing

**Comment by micahstubbs** [comment]
@hypery2k of course. I'll try to get the unit tests passing and ping you when that effort is successful.

**Comment by hypery2k** [comment]
closed in favor to #456

**Pull request closed without merging**